src/views/properties/properties.jsx

```diff
@@ -56,13 +56,19 @@ const makeAuthHeaders = ({ user }) => ({ headers: { 'Authorization': `Bearer ${u
 
 const getDisplayProperties = (properties, showArchived) => properties.filter(p => showArchived || !p.archived);
 
+const formatPropertyManagerNames = (propertyManagers) =>
```

Comment:
Why parens? @Chris-Boe

Reply:
This is pretty standard form and a personal preference as well. When we have a function with no parameters we use empty parentheses, when we have more than one parameter we use parentheses, removing the parentheses only in the case where you have one parameter is somewhat an antipattern, especially for newcomers to coding